### ISSUE TITLE
Default admin user and pass

### ISSUE THREAD

**Issue by renich** [nrpatten/mpos-docker#1]
Hello,

Which are the default admin user and password? Also, how can I change them?

**Comment by nrpatten** [comment]
To ssh into the container run
    docker ps
    docker inspect -f '{{ .NetworkSettings.IPAddress }}' CONTAINER ID
    **ssh dockerip username: root pass: root**

Its in the README.md

**Comment by renich** [comment]
@nrpatten I meant the web interface.

**Comment by jayninja** [comment]
@renich I'm not sure if it is the intended method, but I simply used the signup feature. It complained about not being able to send the email, which is fine, because it still created the user. 

note: I am assuming this is a closed environment, and you are the first and only user.

I attached to the container, and did 'mysql -u root'. 
From there, the mysql is:
use mpos;
update accounts set is_locked=0 ;
update accounts set is_admin=1 where id=1;

Now your user should be able to login as admin.
